pkg/types/compactor.go

```diff
@@ -27,6 +27,8 @@ const (
 	defaultDefragTimeout time.Duration = 8 * time.Minute
 	// defaultSnapshotTimeout defines default timeout duration for taking compacted FullSnapshot.
 	defaultSnapshotTimeout time.Duration = 30 * time.Minute
+	//defaultSleepForInSeconds defines default timeout for sleep command in compaction
+	defaultSleepFor time.Duration = 60 * time.Second
```

Comment:
```suggestion
	defaultSleepFor time.Duration = 0
```

The default behavior should ideally not break any previous behavior of the code. Since there was never a sleep previously, the default for this value should be zero, and should be explicitly set to `60 seconds` by druid when deploying backup-restore, for the gardener usecase.